features/item_creation.feature

```diff
@@ -0,0 +1,18 @@
+#language: fr
+#Louis VAUCHEZ
+#Nicolas HUCHON
+#Jade VENAULT
+#Maxime TRINQUARD
+
+
+Fonctionnalité: Créer un item
+
+Contexte:
+
+Soit le corpus "Vitraux - Bénel" rattaché au portfolio "vitraux"
+Soit "vitraux" le portfolio ouvert
+
+Scénario:
+Soit l'utilisateur "alice" connecté
+Quand l'utilisateur clique sur le bouton de création d'item à coté du corpus "Vitraux - Bénel" pour creer un item "Vitraux Test"
```

Comment:
In the scenario, each step should be related to data, not to the user interface.